Update/omot_010.txt

```diff
@@ -367,7 +367,7 @@ void main()
 //r自分の家庭が壊れてしまったのは、守る努力が足りなかったせい…そう、以前の世界で言っていたような気がする。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "自分の家庭が壊れてしまったのは、守る努力が足りなかったせい…そう、以前の世界で言っていたような気がする。",
-			NULL, "Her home was broken because she didn't put in enough effort to protect it... Right, I believe she said that in a previous world.", Line_Normal);
+			NULL, "Her home was broken because she didn't put in enough effort to protect it... I kind of remember she said that in a previous world.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Her home was broken because she didn't put in enough effort to protect it... I kind of remember her saying that in a previous world.", Line_Normal);
```